packages/rest/src/rest.server.ts

```diff
@@ -260,19 +260,10 @@ export class RestServer extends Context implements Server, HttpServerLike {
    */
   protected _setupOpenApiSpecEndpoints() {
     if (this.config.openApiSpec.disabled) return;
-    // NOTE(bajtos) Regular routes are handled through Sequence.
-    // IMO, this built-in endpoint should not run through a Sequence,
-    // because it's not part of the application API itself.
-    // E.g. if the app implements access/audit logs, I don't want
-    // this endpoint to trigger a log entry. If the server implements
-    // content-negotiation to support XML clients, I don't want the OpenAPI
-    // spec to be converted into an XML response.
```

Comment:
Please preserve this comment, I believe it's still relevant.

Reply:
I did (and will with further refactors), I just moved it to stick with the code that does the actual express binding -- see line 285 in the same file.

The suggested refactors for changing the spec hosting should result in it moving back here.